shaders/position.yaml

```diff
@@ -128,10 +129,11 @@ layers:
         draw:
             polygons:
                 style: buildings
-                order: 4
+                order: function() { return feature.sort_key; }
                 color: [.7,.7,.7]
                 extrude: true
             outline:
+                order: function() { return feature.sort_key + 1; }
```

Comment:
Oh sorry! Yes that makes sense :)  I'm stuck in line-outline-land. Sorry...

On Wed, Mar 9, 2016 at 2:26 PM, Peter Richardson <notifications@github.com>
wrote:

> In shaders/position.yaml
> <https://github.com/tangrams/tangram-docs/pull/73#discussion_r55573825>:
>
> >                  color: [.7,.7,.7]
> >                  extrude: true
> >              outline:
> > +                order: function() { return feature.sort_key + 1; }
>
> All of the examples here are outlines on top of polygons styles, so unless
> I'm misreading this there's no 'inline', just polygon fills. Or do polygon
> outlines work differently than I'm remembering?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/tangrams/tangram-docs/pull/73/files#r55573825>.
>